src/components/app/Overview/Overview.tsx

```diff
@@ -388,6 +395,7 @@ export default function AppOverview({ appMetaInfo, getAppMetaInfoRes, filteredEn
     function renderAppDescription() {
         return (
             <div>
+                {Catalog && <Catalog id={appId} kind={kind} />}
```

Comment:
We should move this to `renderOverviewContent` semantically and the spacing would be properly handled there.